node/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Build Stage
+FROM rust:1.72.1 as build
+COPY . /node/
+WORKDIR /node
+RUN apt-get update && apt-get install -y libclang-dev
+RUN cargo build --release
+RUN cd tools && make node_configs
```

Comment:
make node_configs builds with dev profile which unnecesarily increases the compilation time, since we have already everything built with release profile.